datarequest/datarequest.py

```diff
@@ -3,15 +3,422 @@
 __copyright__ = 'Copyright (c) 2021-2022, Utrecht University'
 __license__   = 'GPLv3, see LICENSE'
 
-from flask import Blueprint, render_template, Response
+from flask import abort, redirect, url_for, send_file, Blueprint, g, jsonify, make_response, render_template, escape, request, session
+
+import os
+import io
+import json
+import api
+
+from irods.message import iRODSMessage
+from werkzeug.utils import secure_filename
 
 datarequest_bp = Blueprint(
     'datarequest_bp',
     __name__,
-    template_folder='templates'
+    template_folder='templates',
+    static_folder='static/datarequest'
 )
 
+# Helper functions
+
+def permission_check(request_id, roles, statuses):
+    return api.call('datarequest_action_permitted', {'request_id': request_id,
+                                                     'statuses': statuses,
+                                                     'roles': roles})['data']
+
+def human_readable_status(request_status):
```

Comment:
May be nicer and more readable to use a Enum, for example: https://github.com/UtrechtUniversity/yoda-ruleset/blob/a0e4cfcf2fb5c6c68902cac3a3967ee8211a017e/util/constants.py#L122